docs/learn/dapp-developer/getting-started.md

```diff
@@ -6,9 +6,20 @@ description: Getting started building dApps on the LUKSO ecosystem.
 
 # Getting started building dApps
 
+## 🖥️ Get the LUKSO Universal Profile browser extension
+
+To build dApps for LUKSO, you will need:
+
+- a Universal Profile (UP)
+- the [LUKSO Universal Profile browser extension](https://chromewebstore.google.com/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn)
```

Comment:
```suggestion
- create a Universal Profile (UP)
- install the [LUKSO Universal Profile browser extension](https://chromewebstore.google.com/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn)
```

Maybe link to `universalprofile.cloud` or `https://docs.lukso.tech/standards/universal-profile/introduction` on the first bullet point?